io.github.eclab.edisyn.yml

```diff
@@ -0,0 +1,45 @@
+
+id: io.github.eclab.edisyn
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.openjdk11
+command: launch.sh
+finish-args:
+    - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+    - --socket=x11
+    - --socket=pulseaudio
+    - --share=ipc
+    - --filesystem=home
```

Comment:
why is home needed?

Reply:
As far as I know there is currently no way to use portals in a Java Swing app. At least not without modifying the code. For clarification: The app has not been designed with flatpak in mind. The original author is not interested in maintaining a flatpak but allowed me to take this job.

Reply:
I will go for the xdg-* solution

Reply:
considered in https://github.com/flathub/flathub/pull/5762/commits/4cb83eed7cc40189ceac5437a087392188ca1ce0